Code/Editor/MainWindow.cpp

```diff
@@ -506,6 +507,12 @@ void MainWindow::Initialize()
             {
                 InitializeMenus();
             }
+
+            m_toolBarManagerInterface = AZ::Interface<AzToolsFramework::ToolBarManagerInterface>::Get();
+            if (m_toolBarManagerInterface)
```

Comment:
question: Should we assert here too? Or is it really okay if this fails gracefully?

Reply:
I think for now it's fine for it to fail gracefully, but I am probably going to have to refactor this when we will switch off the legacy Action Manager so I will rethink this then :)

Reply:
Cool sounds good 👍